src/ocf_stats.c

```diff
@@ -304,8 +306,9 @@ int ocf_core_io_class_get_stats(ocf_core_t core, ocf_part_id_t part_id,
 	stats->dirty_clines = env_atomic_read(&core->runtime_meta->
 			part_counters[part_id].dirty_clines);
 
-	stats->free_clines = cache->conf_meta->cachelines -
-			cache_occupancy_total;
+	max_size = cache->user_parts[part_id].config->max_size;
+	curr_size = ocf_part_get_occupancy(cache, part_id);
+	stats->free_clines = (max_size >= curr_size) ? max_size - curr_size : 0;
```

Comment:
so "free cachelines" now means how far an IO class is from reaching its max? even if cache occupancy is 100%? I think neither approach makes sense...  I think we should only report occupancy for I/O class, "free" counter is meaningless. What do you think?